lib/form_data.js

```diff
@@ -31,7 +31,7 @@ function FormData() {
   this._valueLength = 0;
   this._valuesToMeasure = [];
 
-  CombinedStream.call(this);
+  Object.assign(this, CombinedStream.create(options));
```

Comment:
You can just copy same loop from https://github.com/felixge/node-combined-stream/blob/master/lib/combined_stream.js#L22-L25

Also, I'd keep using `CombinedStream.call(this);` and would add that `options` loop after that.